resources/js/components/ui/form/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from "./Checkbox";
 export * from "./FieldLabel";
+export * from "./SegmentedControl";
```

Comment:
Takie index.ts nam nie psują granularności ładowania tylko potrzebnego kodu?

Reply:
No nie wiem tak szczerze mówiąc. Na razie tak wszędzie mamy, być może należy to przemyśleć ale nie mam do tego kwalifikacji obecnie.